src/vs/workbench/contrib/debug/browser/debugSession.ts

```diff
@@ -913,6 +915,10 @@ export class DebugSession implements IDebugSession {
 		}));
 
 		this.rawListeners.push(this.raw.onDidStop(async event => {
+			if (event.body.reason === 'breakpoint') {
```

Comment:
The current problem with the breakpoints is not that the users are unaware that the breakpoint is hit, but that the breakpoint is on a line.

So the logic should be different.
We need a line change listener, and on each line change to check if a breakpoint is on a line. If it is we play the sound. We will do the same approach with errors.

Reply:
I suggest to introduce a new minimalistic `WorkbenchContribution` just for this

Reply:
Ah, I misunderstood the problem. But https://github.com/microsoft/vscode/issues/135908 is about playing a sound when a breakpoint is hit.

Reply:
@hediet good point. Yeah I then agree we also want this.
I feel like this is mostly requested by non screen reader users. And what I mention is requested by screen reader users.
So in the end we will need both, and the sound we play for those migh be different. Though to start we can play the same sound.